_includes/page-edit.html

```diff
@@ -0,0 +1 @@
+{% if page.title %}<a href="https://github.com/algorithmiaio/dev-center/blob/develop/{{page.path}}" target="_blank" title="Suggest changes to this page" class="edit"><i class="fa fa-edit" aria-hidden="true"></i></a>{% endif %}
```

Comment:
Is a check for `page.title` what one wants here? Or should it be a check for `page.path`?

Reply:
Thx -- page.title is correct. Since the link icon gets rendered after the page
title, it looks really weird on the few pages that have no title... and
those are usually directory-index pages anyway so usu won't need editing.

I guess we could check both... but can page.path ever be null?

Reply:
Appears that it can be overridden by front matter, but won't be null under normal cases.